Gemfile

```diff
@@ -1,2 +1,3 @@
 source "https://rubygems.org"
 gemspec
+gem "kramdown-parser-gfm"
```

Comment:
Without this I wasn't able to start Jekyll locally.
I am not Jekyll/Ruby expert so please advice me if there is a different way to do it!